src/homepageExperience/components/steps/arduino/InitializeClient.tsx

```diff
@@ -47,31 +51,28 @@ export const InitializeClient: FC<OwnProps> = ({
   const me = useSelector(getMe)
   const allPermissionTypes = useSelector(getAllTokensResources)
   const dispatch = useDispatch()
-
+  const url =
+    me.quartzMe?.clusterHost || 'https://us-west-2-1.aws.cloud2.influxdata.com/'
   const currentAuth = useSelector((state: AppState) => {
     return state.resources.tokens.currentAuth.item
   })
+  const token = currentAuth.token
 
   const sortedPermissionTypes = useMemo(
     () => allPermissionTypes.sort((a, b) => collator.compare(a, b)),
     [allPermissionTypes]
   )
+
   const {bucket} = useContext(WriteDataDetailsContext)
 
   useEffect(() => {
     dispatch(getBuckets())
+    dispatch(getAllResources())
```

Comment:
do we need to call `getAllResources()` here? @ChitlangeSahas do you remember why this was called in the initial wizards?

Reply:
We dont have those two dispatch calls in any of the initialize wizards, but because for arduino, you need to select the bucket in initializing step, that's why david might have added it from write data page. I think.

Reply:
So for some reason I thought getAllResources() would get the authorization for the tokens as well as the buckets, but for some reason it only gets the authorization, so I needed to also add in getBuckets() for it to display the buckets and make a token